library/utility/crc.hpp

```diff
@@ -0,0 +1,99 @@
+#pragma once
+
+#include <cstdlib>
+#include <cstdint>
```

Comment:
https://en.cppreference.com/w/cpp/types/is_same

You didn't include <type_traits> (the thing that has std::is_same in it) here. Tidy has caught that the file on its own does not include anything. You get lucky because in the other files, type_traits is probably included somewhere in there.

Reply:
Alright, I'll do this tonight